tests/unit/test_column_noise.py

```diff
@@ -125,9 +119,149 @@ def test_miswrite_zipcodes():
     pass
 
 
-@pytest.mark.skip(reason="TODO")
-def test_miswrite_ages():
-    pass
+def test_miswrite_ages_default_config(dummy_dataset):
+    """Test that miswritten ages are appropriately handled, including
+    no perturbation probabilities defaults to uniform distribution,
+    perturbation probabilities"""
+    config = get_configuration()["decennial_census"]["age"]["age_miswriting"]
+    data = dummy_dataset.rename(columns={"numbers": "age"})["age"]
+
+    # Convert to realistic age
+    maximum_age = 120
+    data = data.apply(pd.to_numeric, args=("coerce",))
+    data = data / data.max() * (maximum_age + 1)
+    data[data.isna()] = -1  # temp nan
+    data = data.astype(int).astype(str)
+    data[data == "-1"] = ""
+    noised_data = _validate_seed_and_noise_data(
+        noise_type=NOISE_TYPES.AGE_MISWRITING, column=data, config=config
+    )
+
+    # Check for expected noise level
+    not_missing_idx = data.index[data != ""]
+    expected_noise = config["row_noise_level"]
+    # todo: Update when generate_incorrect_selection uses exclusive resampling
```

Comment:
Is this todo from something else?  And handling the missing values seems repetitive because we only pass rows we are going to noise to the noise functions but it is a good sanity check.

Reply:
Yes, copy/paste. I'll remove. 

Good point about the noising since you refactored when that happens. Can probably stop checking.